### DISCUSSION TITLE
File Manager - sort by file name

### DISCUSSION THREAD

**Discussion by canpolat** [Zettlr/Zettlr#2650] [Q&A] [answered]
As far as I can see there are two sorting options in File List view: "time" and "name". I was expecting "name" to be the file name, but if I'm not mistaken, it appears to be a mix of filename and document title: Document title takes precedence and if it is not found then the filename is used. I was wondering if there is a way to sort these files by filename (and only by filename)?

**Comment by nathanlesage** [comment, marked as answer]
> it appears to be a mix of filename and document title

Yup, it always sorts by whatever you see, depending on your preferences.

There are two potential fixes in the pipeline:

1. I'll be adding an option to decide what you want to have inside the file lists: Filenames by default, then two checkboxes, one to add `title` properties (checked by default) and then `headings level 1` (off by default).
2. With regard to projects, I'll be implementing an explicit sort order via the frontmatter so that you can force Zettlr to use a specific ordering

**Reply by canpolat** [reply]
Looking forward to the first point
